client/src/api/index.js

```diff
@@ -43,6 +43,17 @@ export const Visitors = {
         '/api/users/all',
         { headers: { Authorization: tkn } },
       );
+    } else if (query.page) {
+      return axios.get(
+        '/api/users/all'
+        , { params: {
+          page: query.page,
```

Comment:
Shouldn't this be called `offset`?

Reply:
I'm using `page` to refer to the page number viewed on the clientside, and using `offset` to refer to the lines of data we are offsetting by. In queries offset is calculated by `page` * `limit` - `limit`.

Reply:
Do you think it will cause confusion with this referring to a different offset than that used in sql?

I think my preference would be to send a page and limit in query, then work out the offset on the server, but happy to rename if you don't think this is an issue.